xml/api-result-0.1.rng

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grammar xmlns="http://relaxng.org/ns/structure/1.0"
+         datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+
+    <start>
+        <ref name="element-pacemaker-result"/>
+    </start>
+
+    <define name="element-pacemaker-result">
+        <element name="pacemaker-result">
+            <attribute name="api-version"> <text /> </attribute>
+            <choice>
+                <element name="installed-devices">
+                    <attribute name="summary"> <text /> </attribute>
```

Comment:
The idea is that it would be the "X devices found" string, but for XML.  So it's this:
```
<installed_devices summary="4 devices found">
```

I'm not sold on it.  It looks kind of cheesy, and maybe we don't care in the XML.  The caller can always count how many children installed_devices has and print that if they care.  Alternately, I could have an attribute that is just the raw number with no text.

Reply:
Let's ditch it -- XML is for nonhumans :) so whatever's processing it can easily count if necessary

Reply:
The raw number would also be reasonable, and we can add that automatically when ending the list if we keep our own count